thriftbp/server.go

```diff
@@ -10,6 +12,10 @@ import (
 	"github.com/reddit/baseplate.go/log"
 )
 
+var (
+	ErrInvalidListenerAddress = errors.New("invalid listener address")
```

Comment:
I don't think this is useful outside this package and would recommend unexporting it.  Exported error sentinels are an API burden.